django/db/backends/oracle/creation.py

```diff
@@ -119,6 +120,39 @@ def _switch_to_test_user(self, parameters):
             self.connection.settings_dict['USER'] = parameters['user']
         real_settings['PASSWORD'] = self.connection.settings_dict['PASSWORD'] = parameters['password']
 
+    def _clone_test_db(self, suffix, verbosity, keepdb):
+        parameters = self._get_cloned_db_params(suffix)
+        settings = self.connection.settings_dict
+        main_login = f"{settings['SAVED_USER']}/{settings['SAVED_PASSWORD']}@{settings['NAME']}"
+        main_test_user = settings['USER']
+        cloned_test_user = parameters['user']
+        with self._maindb_connection.cursor() as cursor:
+            try:
+                self._execute_test_db_creation(cursor, parameters, verbosity, keepdb)
+            except Exception as e:
+                if 'ORA-01543' in str(e):
+                    self._execute_test_db_destruction(cursor, parameters, verbosity)
+                    self._execute_test_db_creation(cursor, parameters, verbosity, keepdb)
+            try:
+                self._create_test_user(cursor, parameters, verbosity, keepdb)
+            except Exception as e:
+                if 'ORA-01920' in str(e):
+                    self._destroy_test_user(cursor, parameters, verbosity)
+                    self._create_test_user(cursor, parameters, verbosity, keepdb)
+            if int(suffix) < 2:
+                exp_cmd = ['expdp',
+                           main_login,
+                           f'schemas={main_test_user}',
+                           f'dumpfile={main_test_user}.dmp'
+                           ]
+                subprocess.run(exp_cmd, shell=False)
```

Comment:
I would use `subprocess.run(..., check=True)` to throw an exception if `expdp` fails. Same for the `impdp` below.

Reply:
The output is already shown by default when it runs. Adding exception handling is a nice touch!